### PR TITLE
Add nginx www path variable

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.2/install.html#
 | `bbb_letsencrypt_email` | E-mail for use with letsencrypt | | _(required when letsencrypt is enabled)_ |
 | `bbb_nginx_privacy` | only log errors not access | `yes` |
 | `bbb_nginx_listen_https` | nginx: use https | `yes` | This is useful for a reverse proxy configuration where the BBB server is behind a load balancing server like haproxy that does SSL termination |
+| `bbb_nginx_root` | Defaut nginx www path of BigBlueButton | `/var/www/bigbluebutton-default` |
```
Comment (BrutalBirdie) [inline README.md:24]: Comment missing for README `| Your Comment |` it's not needed but nice to have.
For example a hint that no trailing `/` is needed and even forbidden.

**Review by BrutalBirdie — APPROVED** [review]
Minor README issue otherwise LGTM.

File: README.md (hunk 1 above)
Comment (anadahz) [reply README.md]: Thank you for reviewing the PR.
Fixed in: https://github.com/ebbba-org/ansible-role-bigbluebutton/pull/107/commits/c5d0db9bbe001d864902bca9374595da8cebb53b

File: tasks/config.yml [hunk 2]
```diff
@@ -290,6 +290,6 @@
   become: true
   copy:
     src: "{{ bbb_custom_presentation }}"
-    dest: "/var/www/bigbluebutton-default/{{ bbb_custom_presentation_name | default(bbb_custom_presentation) }}"
+    dest: "{{ bbb_nginx_root | regex_replace('\\/$', ''}}/{{ bbb_custom_presentation_name | default(bbb_custom_presentation) }}"
```
Comment (francisco-core) [inline tasks/config.yml:293]: I think the `regex_replace` function is missing the closing parenthesis `)`

And just to confirm this regex function is needed to remove the trailing `/`, right?

File: templates/nginx/vhost.conf [hunk 3]
```diff
@@ -26,7 +26,7 @@ server {
 {% endif %}
 
   location / {
-    root   /var/www/bigbluebutton-default;
+    root   {{ bbb_nginx_root | regex_replace('\\/$', '' }};
```
Comment (francisco-core) [inline templates/nginx/vhost.conf:29]: (same) `regex_replace` function is missing the closing parenthesis `)`

**Review by francisco-core — CHANGES_REQUESTED** [review]

File: tasks/config.yml (hunk 2 above)
Comment (anadahz) [reply tasks/config.yml]: Thank you for the review @francisco-core .
I fixed it in: https://github.com/ebbba-org/ansible-role-bigbluebutton/pull/107/commits/bef7c63da4f638dc42669b3fa78460c6bf64fc35

File: templates/nginx/vhost.conf (hunk 3 above)
Comment (anadahz) [reply templates/nginx/vhost.conf]: Fixed in: https://github.com/ebbba-org/ansible-role-bigbluebutton/pull/107/commits/bef7c63da4f638dc42669b3fa78460c6bf64fc35

**Review by francisco-core — APPROVED** [review]

**Review by n0emis — APPROVED** [review]

**Pull request merged**